README.md

```diff
@@ -114,6 +144,21 @@ JSON files are written as an array of objects where each object corresponds to a
  ]
 ```
 
+The equivalent python syntax would be:
```

Comment:
Instead of repeating this under each CSV example and again under each JSON example, let's make this a three subsection things, with `#### CSV` followed by `#### JSON` and finally `#### Python`.